rosbag2_storage/include/rosbag2_storage/metadata_io.hpp

```diff
@@ -12,25 +12,26 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef ROSBAG2_STORAGE__STORAGE_INTERFACES__BASE_INFO_INTERFACE_HPP_
-#define ROSBAG2_STORAGE__STORAGE_INTERFACES__BASE_INFO_INTERFACE_HPP_
+#ifndef ROSBAG2_STORAGE__METADATA_IO_HPP_
```

Comment:
why that change? What's happening here? Why was it necessary to rename that file?

Reply:
This file has not been renamed.
That's just git seing things that are not there...